precompiles/precompile.go

```diff
@@ -561,6 +565,12 @@ func Precompiles() map[addr]ArbosPrecompile {
 		return ArbOwnerImpl.OwnerActs(context, evm, method, owner, data)
 	}
 	_, ArbOwner := MakePrecompile(templates.ArbOwnerMetaData, ArbOwnerImpl)
+	getter = ArbOwner.methodsByName["GetInfraFeeAccount"]
+	setter := ArbOwner.methodsByName["SetInfraFeeAccount"]
+	getter.arbosVersion = 5
```

Comment:
I've realized this won't be committed to the other map. I'm about to open a PR to fix this

Reply:
Fixed in PR #906